kolibri/deployment/default/settings/base.py

```diff
@@ -343,6 +342,19 @@
 
 ENABLE_DATA_BOOTSTRAPPING = True
 
+# Session configuration
+
+SESSION_ENGINE = "django.contrib.sessions.backends.file"
+
+SESSION_FILE_PATH = os.path.join(conf.KOLIBRI_HOME, "sessions")
+
+if not os.path.exists(SESSION_FILE_PATH):
+    if not os.path.exists(conf.KOLIBRI_HOME):
+        raise RuntimeError("The KOLIBRI_HOME dir does not exist")
+    os.mkdir(SESSION_FILE_PATH)
```

Comment:
Hmm, why do we not just use `os.makedirs` here, instead of relying on KOLIBRI_HOME having already been created? Seems risky for fresh installs?

Reply:
Because if you have got this far and KOLIBRI_HOME does not exist then something has already gone wrong.

Reply:
(this gets checked during set_env much earlier in the load cycle).